### PR TITLE
Added Generator Delete Entity By Id

### PR DESCRIPTION
**Pull request by anzin**
**Description** (*)
I've implemented:

1. Generator for the "Delete Entity By Id Command."
2. Added test for "Delete Entity By Id Command'.

Screenshot 2021-02-17 at 15 40 33
Screenshot 2021-02-17 at 15 40 19

**Contribution checklist** (*)
 - [ ] Pull request has a meaningful description of its purpose
 - [ ] All commits are accompanied by meaningful commit messages
 - [ ] All new or changed code is covered with integration/functional tests (if applicable)
 - [ ] All automated tests passed successfully (all builds are green)

### CODE REVIEW

File: resources/META-INF/plugin.xml [hunk 1]
```diff
@@ -238,6 +238,7 @@
         <internalFileTemplate name="Magento PHP Form Generic Button Block Class"/>
         <internalFileTemplate name="Magento Entity New Action Controller Class"/>
         <internalFileTemplate name="Magento New Entity Layout XML"/>
+        <internalFileTemplate name="Magento Delete Entity By Id Command Model"/>
```
Comment (VitaliyBoyko) [inline resources/META-INF/plugin.xml:241]: I wonder why we need the word `Model` here. It's just a command, isn’t it?

Comment (VitaliyBoyko) [reply resources/META-INF/plugin.xml:241]: @eduard13 @coderimus what do you think guys?

Comment (eduard13) [reply resources/META-INF/plugin.xml:241]: 💯 agree, `Model` word is extra

Comment (coderimus) [reply resources/META-INF/plugin.xml:241]: @VitaliyBoyko agree that the `Model` is an extra word.

Comment (VitaliyBoyko) [reply resources/META-INF/plugin.xml:241]: thanks 🙏

**Review by VitaliyBoyko — CHANGES_REQUESTED** [review]
Please remove the `Model` from the command template

**Comment by anzin** [comment]
Hello, @VitaliyBoyko I've fixed the code by recommendation.

Thank you

**Review by VitaliyBoyko — APPROVED** [review]
Cool! ✅

**Pull request merged**